backend/ianalyzer/settings.py

```diff
@@ -20,6 +21,9 @@
 # SECURITY WARNING: keep the secret key used in production secret!
 SECRET_KEY = 'kxreeb3bds$oibo7ex#f3bi5r+d(1x5zljo-#ms=i2%ih-!pvn'
 
+if SENTRY_DSN := os.getenv('SENTRY_DSN'):
+    sentry_sdk.init(dsn=SENTRY_DSN, tracing=True)
```

Comment:
I cannot find the `tracing` option in the [documentation](https://docs.sentry.io/platforms/python/configuration/options/). What does it do? If this means we'll go as deep as tracing SQL queries, I wouldn't think this is ncessary.